api/bootstrap/kubeadm/v1beta2/kubeadm_types.go

```diff
@@ -164,6 +164,20 @@ type ClusterConfiguration struct {
 	// featureGates enabled by the user.
 	// +optional
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
+
+	// certificateValidityPeriodDays specifies the validity period for a non-CA certificate generated by kubeadm.
+	// If not specified, kubeadm will use a default of 3650 days (10 years).
+	// +optional
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Is zero a valid value for this field? What would an explicit zero mean?

Reply:
Apologies, Its supposed to be 1.

Reply:
Updated.